app/feature/feature-claim-chat/src/commonMain/kotlin/com/hedvig/feature/claim/chat/ui/ClaimChatDestination.kt

```diff
@@ -500,21 +492,32 @@ private fun DeflectStep(
       navigateToDeflect(stepId, deflect)
     }
   }
-  HedvigNotificationCard(
-    message = text,
-    priority = NotificationDefaults.NotificationPriority.InfoInline,
-    style = NotificationDefaults.InfoCardStyle.Button(
-      buttonText = stringResource(Res.string.important_message_read_more),
-      onButtonClick = { navigateToDeflect(stepId, deflect) },
-    ),
-    modifier = modifier,
-  )
+  if (text != null) {
+    HedvigNotificationCard(
+      message = text,
+      priority = NotificationDefaults.NotificationPriority.InfoInline,
+      style = NotificationDefaults.InfoCardStyle.Button(
+        buttonText = stringResource(Res.string.important_message_read_more),
+        onButtonClick = { navigateToDeflect(stepId, deflect) },
+      ),
+      modifier = modifier,
+    )
+  }
+
 }
 
 @Composable
-private fun TaskStep(itemText: String, taskContent: StepContent.Task, modifier: Modifier = Modifier) {
+private fun TaskStep(
+  itemText: String?,
+  taskContent: StepContent.Task,
+  modifier: Modifier = Modifier,
+) {
   Column(modifier) {
-    HedvigText(itemText)
+    itemText?.let {
+      HedvigText(
+        itemText,
+      )
+    }
     if (taskContent.descriptions.isNotEmpty()) {
       Column {
         Spacer(Modifier.height(8.dp))
```

Comment:
Here the spacer above the descriptions might not be intended if the text isn't present. Perhaps the spacer should be moved inside the itemText null check so it's only there if the text is also there, like we do for some other cases further down?